### ISSUE TITLE
[BUG] Error updating Global credentials of DynamoDB 

### ISSUE THREAD

**Issue by fahimmahmoodmir** [dynamoose/dynamoose#964]
### Summary:
While updating the credentials of a dynamoose via `dynamoose.aws.sdk.config.update()` as described in [documentation](https://dynamoosejs.com/getting_started/Configure#global), application crashes with `TypeError: dynamoose.aws.sdk.config.update is not a function` error. Which while trying to debug could be seen that it is an object property. Although a function call also exists under proto. Do I need to do something different?

![image](https://user-images.githubusercontent.com/2732245/87165865-02d6c280-c2cb-11ea-880f-1eab37b0811c.png)

I mean i can try to use `dynamoose.aws.sdk.config.constructor.prototype.update` method but in the documentation, this is not defined so I want to be sure before using `dynamoose.aws.sdk.config.constructor.prototype.update` that if there is a clean way to do it which is not defined in the documentation.

#### General
```
const dynamoose = require('dynamoose');

// I have values for <AWS_ACCESS_KEY_ID>, <AWS_SECRET_ACCESS_KEY> and <AWS_REGION> so ignore here on bug report snipit 

  dynamoose.aws.sdk.config.update({
    credentials: {
      accessKeyId: <AWS_ACCESS_KEY_ID>,
      secretAccessKey: <AWS_SECRET_ACCESS_KEY>
    },
    region: <AWS_REGION>
  });
```

While debugging I can see that `update` exists on `dynamoose.aws.sdk.config` as property and also as a function under prototype. Thus when trying to console log with `typeof dynamoose.aws.sdk.config.update` it's coming as an object. 

### Current output and behaviour (including stack trace):
When i am trying to update the configuration of dynamooose i receive `TypeError: dynamoose.aws.sdk.config.update is not a function`

![image](https://user-images.githubusercontent.com/2732245/87163111-2435af80-c2c7-11ea-89a3-25ee07b40226.png)

### Expected output and behaviour:
Configuration function is called and configurations are updated as well.

### Environment:

Operating System: macOS (Mojave)
Operating System Version: 10.14.6 (18G4032)
Node.js version (`node -v`): v12.13.1
NPM version: (`npm -v`): 6.14.5
Dynamoose version: 2.2.1

### Other:
- [x] I have read through the Dynamoose documentation before posting this issue
- [x] I have searched through the GitHub issues (including closed issues) and pull requests to ensure this issue has not already been raised before
- [x] I have searched the internet and Stack Overflow to ensure this issue hasn't been raised or answered before
- [x] I have tested the code provided and am confident it doesn't work as intended
- [x] I have filled out all fields above
- [x] I am running the latest version of Dynamoose

**Comment by fishcharlie** [comment]
@fahimmahmoodmir I can't seem to reproduce this.

```
charliefish@Charlies-MacBook-Pro tmp3 % node
Welcome to Node.js v12.13.1.
Type ".help" for more information.
> const dynamoose = require("dynamoose");
undefined
> dynamoose.aws.sdk.config.update({
... 	credentials: {
..... 		accessKeyId: "KEY",
..... 		secretAccessKey: "SECRET"
..... 	},
... 	region: "us-west-2"
... });
undefined
```

No errors were thrown, and everything worked as expected.

It almost looks like you are setting `dynamoose.aws.sdk.config.update` equal to something. Like `dynamoose.aws.sdk.config.update = {}`, which is overwriting the method.

When I run the following I get a function:

```
> dynamoose.aws.sdk.config.update
[Function: update]
```

I'm assuming if you ran the same you'd get an object.

Can you double check your code to ensure that you aren't overwriting that function anywhere? I'm pretty sure that is what is happening. If you aren't, can you provide a zip file or something of a [MCVE](https://github.com/dynamoose/dynamoose/blob/master/CONTRIBUTING.md#minimal-complete-and-verifiable-example-mcve) so I can debug further?

**Comment by fahimmahmoodmir** [comment]
Ok I will do some digging in our code and will let you know my findings

**Comment by fahimmahmoodmir** [comment]
Hi, I figured out the problem. 
This is in one place of our code we were globally updating the aws-sdk package credentials
```
  AwsSdk.config.update = {
    credentials: {
      accessKeyId: <AWS_ACCESS_KEY_ID>,
      secretAccessKey: <AWS_SECRET_ACCESS_KEY>
    },
    region: <AWS_REGION>
  };
```

later in another part of code, the code was again initialized with the dynamoose credentials as I showed above. So it seems that overwriting the value of `AwsSdk.config.update` created this trouble as I think it could be the case that dynamoose use the same aws-sdk `config.update` property. I found that aws-sdk also provide the update function for the setting property. Thus changing our aws-sdk credentials like following resolved issue :

```
  AwsSdk.config.update({
    credentials: {
      accessKeyId: AWS_ACCESS_KEY_ID,
      secretAccessKey: AWS_SECRET_ACCESS_KEY
    },
    region: AWS_REGION
  });
```
